libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -250,3 +251,8 @@ def ExecuteRequest(client: 'googleapiclient.discovery.Resource',
     next_token = response.get('nextPageToken')
     if not next_token:
       return responses
+
+
+def FormatRFC3339(datetime_instance: datetime.datetime) -> str:
+  """Formats a datetime per RFC 3339."""
```

Comment:
An example of what the formatting looks like here would be great.

Reply:
Done